packages/paste-core/layout/media-object/__fixtures__/CustomizableMediaBody.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import {MediaBody} from '../src';
+import type {MediaBodyProps} from '../src';
+
+export interface CustomizableMediaBodyProps extends MediaBodyProps {
+  element?: string;
+  variant?: 'primary' | 'secondary';
```

Comment:
This might be jumping ahead since you'll be discussing this during show and tell tomorrow, but what is it the variant is doing here? I don't see it present in the previous version of the Media Object component.

Reply:
Ah, is this to make it easier to create different stories? Is that also the reason we made these 'Customizable' components, for the stories?

Reply:
We're using variant here to create new variants of MediaObject. With the CustomizationProvider, consumers will be able to create their own variants of Paste components.

And yeah, this file just makes it easier to use the same custom MediaObject for stories and tests.

Reply:
Actually we're going to remove variant for this round. We only want to expose the existing component variants if there are any.